### PR TITLE
[MB-5580] Remove text-indent css prop from queue status multiselect

### PR DESCRIPTION
**Pull request by duncan-truss**
## Description

The multiselect status filter options in the TOO and TIO queues were being obscured by the checkbox input.  I removed the text-indent styling on the checkbox input label which renders the label after the checkbox.

## Reviewer Notes

I tested on Firefox, Chrome, and Safari and it visually looks the same cross browser.

The accessibility needs improved on this element, I couldn't tab down into the options like I would have expected.

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make server_run
make office_client_run
```

1. Login in as either the TOO or the TIO office user.
2. Click on the status filter select to reveal the menu of options.
3. Check that the label no longer overlaps with the checkbox input.

## Code Review Verification Steps

* [ ] If the change is risky, it has been tested in experimental before merging.
* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#logging)
* [ ] The requirements listed in
 [Querying the Database Safely](https://github.com/transcom/mymove/wiki/Backend-Programming-Guide#querying-the-database-safely)
 have been satisfied.
* Any new migrations/schema changes:
  * [ ] Follow our guidelines for zero-downtime deploys (see [Zero-Downtime Deploys](https://github.com/transcom/mymove/wiki/migrate-the-database#zero-downtime-migrations))
  * [ ] Have been communicated to #g-database
  * [ ] Secure migrations have been tested following the instructions in our [docs](https://github.com/transcom/mymove/wiki/migrate-the-database#secure-migrations)
* [ ] There are no aXe warnings for UI.
* [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/master/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
* [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-5580) for this change

## Screenshots

Screen Recording 2020-11-24 at 10 00 27 AM mov

Screen Shot 2020-11-24 at 10 06 02 AM

### CODE REVIEW

**Review by amanda5iu — APPROVED** [review]
Ahh, that's better :)

**Review by noelledusahel — APPROVED** [review]
lgtm - just one comment below

File: src/components/Table/Filters/MultiSelectCheckBoxFilter.module.scss [hunk 1]
```diff
@@ -32,14 +32,14 @@
...
+      outline: 0.25rem solid #2491ff !important;
       outline-offset: 0;
     }
 
     .MultiSelectCheckBoxFilter__menu {
-      @include u-margin-top(.5);
+      @include u-margin-top(0.5);
       @include u-radius(0);
-      max-width: 183px;
+      //max-width: 183px;
```
Comment (noelledusahel) [inline src/components/Table/Filters/MultiSelectCheckBoxFilter.module.scss:42]: is there a reason to leave this commented rather than removing it?

**Review by suzubara — APPROVED** [review]

File: src/components/Table/Filters/MultiSelectCheckBoxFilter.module.scss (hunk 1 above)
Comment (duncan-truss) [reply src/components/Table/Filters/MultiSelectCheckBoxFilter.module.scss:42]: The comment had no effect actually, I will put it back before merging

Comment (duncan-truss) [reply src/components/Table/Filters/MultiSelectCheckBoxFilter.module.scss]: Actually it is worth deleting, when you select multiple statuses the filter grows in width so the dropdown menu shouldn't be artificially smaller.

**Review by donaldthai — APPROVED** [review]
Thanks for fixing this! 🚀

**Pull request merged**